longshot/application/main.tf

```diff
@@ -0,0 +1,218 @@
+variable "pipeline" {}
```

Comment:
maybe add a header comment saying this is a template and you expect instantiations of this template to initialize variables you listed in the beginning and didn't assign any value.  It might make sense to group all the unassigned values together, to help the reader out. Also might be good to separate required and optional (with comments)

Reply:
Good idea! I've updated documentation for these in #55.